bazel/repositories.bzl

```diff
@@ -217,6 +217,15 @@ def cartographer_repositories():
       ],
   )
 
+  _maybe(native.http_archive,
+      name = "prometheus_cpp",
```

Comment:
Please use com_github_jupp0r_prometheus_cpp to follow convention.

Reply:
Done